agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -43,6 +43,7 @@ import (
 	"github.com/cihub/seelog"
 	"github.com/docker/docker/api/types/volume"
 	docker "github.com/fsouza/go-dockerclient"
+	"github.com/docker/docker/api/types"
```

Comment:
nit: `"github.com/docker/docker/api/types"` should be grouped together with `"github.com/docker/docker/api/types/volume"`